### ISSUE TITLE
npm install failed

### ISSUE THREAD

**Issue by gabx** [ether-camp/ethereum-testing-reference#1]
ethereum-testing-reference % npm install

npm WARN starter@ No repository field.
npm WARN starter@ No license field.
npm ERR! Linux 4.6.1-2-hortensia
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v6.2.2
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE

npm ERR! secp256k1@3.1.0 install: `npm run rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the secp256k1@3.1.0 install script 'npm run rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the secp256k1 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs secp256k1
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls secp256k1
npm ERR! There is likely additional logging output above.

[npm-debug.txt](https://github.com/ether-camp/ethereum-testing-reference/files/347320/npm-debug.txt)

**Comment by romanman** [comment]
@gabx : more info on your environment please.


@asinyagin

**Comment by asinyagin** [comment]
@gabx yeah, there's a problem in secp256k1 build-script. It tries to use `/usr/bin/node`, but ubuntu installs nodejs to `/usr/bin/nodejs`. As a workaround you can run `ln -s /usr/bin/nodejs /usr/bin/node`.

https://github.com/cryptocoinjs/secp256k1-node/issues/88

**Comment by romanman** [comment]
@gabx is it ok now?

**Comment by gabx** [comment]
Sorry for late answer. I run Linux Fedora 24. I do not have `/usr/bin/nodejs` but rather `/usr/bin/node`
% npm version
{ npm: '3.10.3',
  ares: '1.10.1-DEV',
  http_parser: '2.7.0',
  icu: '57.1',
  modules: '48',
  node: '6.2.2',
  openssl: '1.0.2h',
  uv: '1.9.1',
  v8: '5.0.71.52',
  zlib: '1.2.8' }

**Comment by asinyagin** [comment]
@gabx i've just tried to run the tests on fedora 24 docker and haven't got the error. I see you have different nodejs version, how did you install it?

**Comment by romanman** [comment]
@gabx @asinyagin  ::: should we close that one ?
